tests/quick_tests/CMakeLists.txt

```
@@ -66,11 +66,11 @@ MACRO(make_quicktest test_basename build_name mpi_run)
 
   # this is a hack to make sure the -OK file is deleted
   # even if compilation fails.
-  ADD_CUSTOM_TARGET(kill-${_target}-OK
-        COMMAND rm -f ${_target}-OK
+  ADD_CUSTOM_TARGET(reset-${_target}-OK
+    COMMAND ${CMAKE_COMMAND} -E rm -f ${_target}-OK
```

It needs to be this:
```suggestion
    COMMAND ${CMAKE_COMMAND} -E remove -f ${_target}-OK
```

---

I will ad a version check and fall back to `remove` for cmake versions prior to 3.17.

---

Is `remove` deprecated? If not, why not always use it?

---

@bangerth It got deprecated in favor of `rm` in version 3.17. I will switch to `remove` for the time being because we already have it in our codebase a couple of times.